marklogic-data-hub-central/ui/src/components/modeling/property-table/property-table.tsx

```diff
@@ -55,7 +55,7 @@ const DEFAULT_SELECTED_PROPERTY_OPTIONS: PropertyOptions = {
   multiple: '',
   pii: '',
   sort: false,
-  facet: false,
+  facetable: false,
   wildcard: false
```

Comment:
No need to change right now - but @brucean52  is "wildcard" used? I didn't think we were supporting that. Enabling a word lexicon alone won't enabling wildcarded search. I'd rather see this removed (separate JIRA ticket) if we're not doing anything with it.

Reply:
yeah, wildcard is hooked up to wordLexicon right now

Reply:
Can you work with Allison on changing this? We can't release this, because wordLexicon alone will not enable wildcard search.